### DISCUSSION TITLE
How to filter collected log files by pod label in kuberneteslogs?

### DISCUSSION THREAD

**Discussion by guidao** [vectordotdev/vector#21628] [General]
For example, only collect logs from pods that have either the .kubernetes.podlabels.service or .kubernetes.podlabels.unit labels

**Comment by jszwedko** [comment]
I think https://vector.dev/docs/reference/configuration/sources/kubernetes_logs/#extra_label_selector is what you want.

**Reply by guidao** [reply]
But label selector seems to support only “and” but not “or” semantics.

**Reply by jszwedko** [reply]
Ah, you are right. You could consider adding a new label that you apply to pods you'd like to collect.

**Reply by guidao** [reply]
I think a better way would be to get the label info for the pods and then filter by vrl, since I can't just add labels to other people's pods. Is there a possibility to support this feature?
